src/gatsby-plugin-theme-ui/components.js

```diff
@@ -141,6 +142,18 @@ const RepoButton = ({ url, host = new URL(url).host }) => {
           <span>GitLab</span>
         </Link>
       )
+    case 'bitbucket.org':
+      return (
+        <Link
+          href={url}
+          target="_blank"
+          rel="noopener noreferrer"
+          variant="RepoButton"
+        >
+          <BitbucketIcon width="14" height="14" />
```

Comment:
why different size?

```suggestion
          <BitbucketIcon width="18" height="18" />
```

Reply:
<img width="60" alt="image" src="https://user-images.githubusercontent.com/414967/200359923-2ccdc954-2e3d-4ffb-8790-667a40bf2adf.png">
<img width="53" alt="image" src="https://user-images.githubusercontent.com/414967/200360021-f9437935-e551-42f5-8f8d-dea9ca413575.png">
<img width="43" alt="image" src="https://user-images.githubusercontent.com/414967/200360127-df008e0f-51f7-43bc-b56d-b991e700a475.png">

vs 

<img width="50" alt="image" src="https://user-images.githubusercontent.com/414967/200360223-9bdbc349-08ec-4fb2-9829-485bf7f1742a.png">

Reply:
It does not really matter to have its own size. In the  page all the values are harcoded according to the needs